dts/arm/st/g4/stm32g474.dtsi

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2019 Richard Osterloh <richard.osterloh@gmail.com>
```

Comment:
Pleae use your copyright.

Reply:
Can't just delete Richard's copyright, but you can add Francois'.